app/data_providers/dispatch.py

```diff
@@ -59,7 +59,9 @@ async def get_ranked_corpus_slate(self, user: UserIds, recommendation_count: int
             logging.info(f'SetupMoment is unpersonalized for user {user.user_id} because no preferences were found.')
             topics = await self.topic_provider.get_topics(self.DEFAULT_TOPICS)
 
-        items = rank_by_preferred_topics(items, topics, recommendation_count)
+        random.shuffle(items)
+        items = spread_topics(items)
+        items = rank_by_preferred_topics(items, topics)
```

Comment:
SetupMoment is deprecated and not used, so it doesn't matter what we do here, really. I'll check with Clients and remove this endpoint in a future PR.